README.md

```diff
@@ -2,84 +2,88 @@
+  https://virtocommercedemo1.blob.core.windows.net/catalog/1428965138000_1133723.jpg will be
 ```
 Thumbnail name is going to be
 ```
   https://virtocommercedemo1.blob.core.windows.net/catalog/1428965138000_1133723_large.jpg
 ```
 
-* Width: Thumbnail width in pixels
-* Height: Thumbnail height in pixels
-* Resize method.
+* **Width**: Thumbnail width in pixels
+* **Height**: Thumbnail height in pixels
+* **Resize method**.
 
 # Resize method options
 
 **1. "FixedSize" method.**
 
-Resizes the origingal image to the desired size while maintaining the aspect ratio without cropping. If aspect ratio of original image and the thumbnail doesn't match then white space will be filled with color of background.
+This method resizes the original image to the desired size while maintaining the aspect ratio without cropping. If the aspect ratio of the original image and the thumbnail don't match, the white space will be colored with the color of the background.
 
 **2. "FixedHeight" method.**
 
-Resizes the image to the desired size while maintaining the aspect ratio without cropping. Width of thumbnail will be calculated according aspect ratio. 
+Resizes the image to the desired size while maintaining the aspect ratio without cropping. The width of thumbnail will be calculated according to the aspect ratio.
 
 **3. "FixedWidth" method.**
 
-Resizes the image to the desired width while maintaining the aspect ratio.  If aspect ratio of original image and the thumbnail doesn't match then white space will be filled with color of background.
+Resizes the image to the desired width while maintaining the aspect ratio.  If the aspect ratio of the original image and the thumbnail don't match , then the white space will be colored with the background color.
 
 **4. "Crop" method.**
 
-Resize the image until one of the sides will match to the given dimensions while maintaining the aspect ratio. Part of the image will be cut off. The thumbnail will be resized and cropped the anchor position. You can specify an anchor position in the o'sption Anchor Position: TopLeft, TopCenter, TopRight, CenterLeft, Center, CenterRight, BottomLeft, BottomCenter, BottomRight.
+This method resizes the image until one of the sides matches the given dimensions while maintaining the aspect ratio. A part of the image will be cut off. The thumbnail will be resized and cropped according to the anchor position. You can specify an anchor position in the o'sption Anchor Position: TopLeft, TopCenter, TopRight, CenterLeft, Center, CenterRight, BottomLeft, BottomCenter, BottomRight.
 
 Now you can use the two thumbnails wherever you want to display a smaller version of the original image.
 You can do it, just add a needed suffix (grande or medium) to product image url.
 
-# Running the task
+## Running the task
 
 You can select multiple task and run them in the task list or run just one task from the task blade.
```

Comment:
multiple task**s**

Reply:
`## Running the task
is ok. What I meant was "s" missing in "You can select multiple task and"